app/containers/message/Message.js

```diff
@@ -63,11 +63,15 @@ MessageInner.displayName = 'MessageInner';
 const Message = React.memo((props) => {
 	if (props.isThreadReply || props.isThreadSequential || props.isInfo) {
 		const thread = props.isThreadReply ? <RepliedThread {...props} /> : null;
+		const { username } = props.author;
 		return (
 			<View style={[styles.container, props.style]}>
 				{thread}
 				<View style={[styles.flex, styles.center]}>
 					<MessageAvatar small {...props} />
+					<Text style={styles.username}>
```

Comment:
Try `flex: 1` or set `width` of text and it'll `ellipsis` programmatically, you can remove this logic with substring.

Reply:
Thanks for the advice, by limiting the text width and setting `numberOfLines` to 1 all the info messages are know aligned due to the author name having the same size.